back/app/controllers/web_api/v1/events_controller.rb

```diff
@@ -64,9 +64,22 @@ def event_params
       :project_id,
       :start_at,
       :end_at,
+      :location_description,
+      location_point_geojson: [:type, { coordinates: [] }],
       location_multiloc: CL2_SUPPORTED_LOCALES,
       title_multiloc: CL2_SUPPORTED_LOCALES,
       description_multiloc: CL2_SUPPORTED_LOCALES
-    )
+    ).tap do |p|
+      # Set default location_description if not provided and location_multiloc is
+      # provided. This will be removed once the location_multiloc parameter is removed.
+      next if p[:location_multiloc].blank?
+
+      p[:location_description] ||= p.dig(:location_multiloc, default_locale)
+      p[:location_description] ||= p[:location_multiloc].values.first
```

Comment:
I would have expected this to be covered in the model rather than in the controller. But given it's temporary in any case, fine with it.

Reply:
I actually agree. Thought about while writing it, but stick to this solution since it's temporary.